packages/fluentui/docs/src/components/Playground/renderConfig.ts

```diff
@@ -30,34 +30,45 @@ type CodeSandboxImport = {
   required: boolean;
 };
 
+const newestNightlyReleaseUrlPrefix = `https://fluentsite.blob.core.windows.net/nightly-builds/newest`;
+
+const getPackageVersion = (packageName: string, latestVersion: string) => {
+  // CI build tarballs from `npm pack` and publish them nightly to blob storage for fluent ui packages
+  // When building nightly release docsite, use these urls instead of the latest package version
+  if (process.env.NIGHTLYRELEASE === 'true') {
```

Comment:
```suggestion
  if (process.env.NIGHTLYRELEASE) {
```

I don't expect that env variable will be present at all in this case

Reply:
I have a step that does this for non-nightly docsites: `echo "##vso[task.setvariable variable=nightlyRelease]false"`.
I tried to make NIGHTLYRELEASE to undefined but it didn't work. It's always empty string